reference/price_lists.v3.yml

```diff
@@ -2681,7 +2706,7 @@ paths:
         - $ref: '#/components/parameters/ChannelIdInParam'
       responses:
         '204':
-          description: An empty response.
+          description: No Content.
```

Comment:
Delete returns a `204 No Content`, but it does not return a body along with it. So the ref to `$/components/schemas/NoContent` can be removed.

Reply:
kk, good catch, I removed the unneeded response body